python/plugins/grassprovider/grass_provider.py

```diff
@@ -180,5 +180,5 @@ def canBeActivated(self):
 
     def tr(self, string, context=''):
         if context == '':
-            context = 'Grass7AlgorithmProvider'
+            context = 'GrassProvider'
```

Comment:
won't this force translators to re-translate everything? If so, it's an internal hidden detail and I don't think we should change it.

Reply:
Translation context's rename reverted.

Reply:
Anyway, looking at https://github.com/qgis/QGIS/pull/56196, it seems to me the translation context has been already implicitly renamed in grass_algorithm.py, since the Grass7Algorithm class was renamed GrassAlgorithm and the translation context is based on the class name
```python
    def tr(self, string, context=''):
        if context == '':
            context = self.__class__.__name__
```